planning/behavior_path_planner/include/behavior_path_planner/util/lane_change/lane_change_module_data.hpp

```diff
@@ -79,6 +79,11 @@ struct LaneChangeTargetObjectIndices
   std::vector<size_t> target_lane{};
   std::vector<size_t> other_lane{};
 };
+
+enum class LaneChangeType {
+  Normal = 0,
+  ExternalRequest = 0,
```

Comment:
Is this suggestion right?

```suggestion
  ExternalRequest = 1,
```